pkg/capacity/capacity.go

```diff
@@ -720,13 +733,20 @@ func (c *Controller) isObsolete(capacity *storagev1alpha1.CSIStorageCapacity) bo
 	return true
 }
 
-// isControlledByUs implements the same logic as https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1?tab=doc#IsControlledBy,
+// isOwnedByUs implements the same logic as https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1?tab=doc#IsControlledBy,
 // just with the expected owner identified directly with the UID.
-func (c *Controller) isControlledByUs(owners []metav1.OwnerReference) bool {
-	for _, owner := range owners {
+func (c *Controller) isOwnedByUs(capacity *storagev1alpha1.CSIStorageCapacity) bool {
+	for _, owner := range capacity.OwnerReferences {
 		if owner.Controller != nil && *owner.Controller && owner.UID == c.owner.UID {
 			return true
 		}
 	}
 	return false
 }
+
+// isManaged checks the labels to determine whether this capacity object is managed by
+// the controller instance.
```

Comment:
Since the informer is already filtered by labels in the next commit, and all callers of `isManaged()` fetch from the informer, is this mostly here as a safeguard? If so, nit: could you add a comment to clarify?

Reply:
Correct. I added a comment.